src/components/AddBook.jsx

```diff
@@ -25,22 +25,18 @@ const AddBook = () => {
 
   return (
     <div className="newbook">
-      <h3>ADD BOOK</h3>
+      <h3>ADD NEW BOOK</h3>
       <form onSubmit={submitHandler}>
         <input
           type="text"
           value={title}
           onChange={(e) => setTitle(e.target.value)}
-          placeholder=" New Book"
-          required
-        />
-        <input
-          type="text"
-          value={author}
-          onChange={(e) => setAuthor(e.target.value)}
-          placeholder=" Add Author"
+          placeholder="Book Title"
           required
         />
+        <select>
+          <option value="" hidden selected>Category</option>
+        </select>
         <button type="submit">Add Book</button>
       </form>
     </div>
```

Comment:
- [ ] As required for the project to **EXACTLY** match the `Zeplin` design, kindly ensure that you use the same `fonts(e.g.: family, weight, size,...)`, `blue color`, `spacing`, like the oned used on the template, `the required input`,...
 ### Template design
![Screenshot from 2023-06-23 02-37-06](https://github.com/Katended/book-store/assets/83042742/20ff4487-d44a-45e7-94e6-23f4c078f985)
### Your design
![Screenshot from 2023-06-23 02-41-16](https://github.com/Katended/book-store/assets/83042742/a0148c81-73f2-4a0c-a2f2-861a20e02e29)